examples/international_address_autocomplete.js

```diff
@@ -4,42 +4,47 @@ const Lookup = SmartySDK.internationalAddressAutocomplete.Lookup;
 
 // US Autocomplete Pro only supports using Embedded Keys
 let key = process.env.SMARTY_EMBEDDED_KEY;
-const credentials = new SmartyCore.SharedCredentials(key);
+let hostname = process.env.SMARTY_HOSTNAME;
+const credentials = new SmartyCore.SharedCredentials(key, hostname);
```

Comment:
usually you won't be passing in the hostname because the browser would be setting it in the request automatically. We only have it as a param on the `SharedCredentials` class for testing purposes so we can pass in a mock hostname.

The assumption with this example file is that you would be copying and pasting this code into your website code and then running it there. We can simulate that locally for our own troubleshooting/testing purposes by creating an html file that loads this code and run on localhost.

Reply:
Fixed